OutOfSchool/OutOfSchool.WebApi/Controllers/V1/PersonalInfoController.cs

```diff
@@ -42,17 +42,14 @@ public class PersonalInfoController : ControllerBase
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<IActionResult> GetPersonalInfo()
     {
-        ShortUserDto info;
+        var info = null as ShortUserDto;
+
         if (currentUserService.IsInRole(Role.Parent))
         {
             info = await parentService.GetPersonalInfoByUserId(currentUserService.UserId);
         }
```

Comment:
(Code style, IMHO, don't insist on changes) Please consider usage of ternary operator here, instead of defining variable, setting it to `null` and then immediately in next line conditionally setting it to other value.

My preferred way to write such code is:
```
        var info = currentUserService.IsInRole(Role.Parent)
            ? await parentService.GetPersonalInfoByUserId(currentUserService.UserId)
            : null;
        info ??= await userService.GetById(currentUserService.UserId);
``` 
Benefits:
- you don't need that weird `null as ShortUserDto`;
- you immediately see that `if` is needed only for setting the same variable;
- less code.